src/react-to-webcomponent.ts

```diff
@@ -29,6 +29,27 @@ var define = {
   },
 }
 
+interface R2WCOptions {
+  shadow?: string | boolean
+  dashStyleAttributes?: boolean
+}
+
+interface React {
```

Comment:
You should be able to just use the React and ReactDOM types that come from React/ReactDOM. (Though it might be weird since we support both ReactDOMs.) If you can, then you can use React.Component down on line 64, instead of `{ propTypes?: object }`.

Reply:
React isn't a dependency in this project (except in the individual react versions in the test folders) aaand I don't think we could make multiple versions a dep